ML-Frameworks/pytorch-aarch64/dockerize.sh

```diff
@@ -35,12 +35,20 @@ if ! [ -e "$1" ] || ! [ -e "$2" ]; then
     exit 1
 fi
 
+# Grab files to be copied into the container
```

Comment:
I think it's a good idea to include these md files in the built image, but I'm going to be honest I don't like this copying, because it makes it even harder to build the docker image without this script. Also, we will end up deleting the `SECURITY.md` which _should_ be in this directory. Also, is this license the correct one? Shouldn't the license of the built image include licenses of all the sources too?